src/cordova/apphost.js

```diff
@@ -227,7 +167,7 @@ define(['appStorage', 'browser'], function (appStorage, browser) {
         moreIcon: 'dots-vert',
         getSyncProfile: getDeviceProfile,
         getDefaultLayout: function() {
-            // FIXME: Needs Implementation!
+            return 'mobile';
```

Comment:
This change already landed in `master` right? Perhaps something is off with the git history in your branch?

Reply:
Well I merged the branch from that fix with master, then I created another branch for the subtitles fix. That is the merge to master. Should I proceed differently?

Reply:
I forget to merge from upstream. It solved now.